src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+
+import { Person } from '../../types/Person';
+import { PersonList } from '../PersonList';
+import './Autocomplete.scss';
+
+type Props = {
+  people: Person[],
+  onSelect: (person: Person) => void,
+  delay: number,
+};
+
+export const Autocomplete: React.FC<Props> = ({
+  people,
+  onSelect,
+  delay,
+}) => {
+  const [isListVisible, setIsListVisible] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const [debauncedInputValue, setDebauncedInputValue] = useState(inputValue);
```

Comment:
There's a typo in the word 'debaunced'. It should be 'debounced'. This won't cause any errors, but it's best to fix it for readability and maintainability.